cfgov/v1/util/util.py

```diff
@@ -15,14 +17,20 @@ def to_camel_case(snake_str):
     components = snake_str.split('_')
     return components[0] + "".join(x.title() for x in components[1:])
 
+
+def get_unique_id(prefix='', suffix=''):
+    index = hex(int(time()*10000000))[2:]
```

Comment:
Yep, same for me when the floating point number is converted into a string. 

`str(time.time())` produces `1451585483.05`